pkgs/development/tools/xcbuild/developer.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, platform, toolchain, xcbuild, writeText }:
+
+stdenv.mkDerivation {
+  name = "Xcode.app";
```

Comment:
I'd call it a "developer root" of some kind. "Xcode.app" includes a developer root but also the IDE and such, so might be less clear.